lib/fog/image/openstack/v2/models/image.rb

```diff
@@ -162,6 +171,42 @@ def remove_tag(tag)
             requires :id
             service.remove_tag_from_image(id, tag)
           end
+
+          private
+
+          def build_update_json_patch(client_attributes, server_attributes)
```

Comment:
I am fine with this now, but d oyou think you could extract all below methods to some helper for PATCH, these methods will be reusable elsewhere.